slim/datasets/cifar100.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
This file was copied from cifar10.py.  We could instead handle both in a new file cifar.py, but I figured it's a short file and separating it makes it easier to tweak stuff if needed.